modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSqlOperatorsTest.java

```diff
@@ -326,6 +331,25 @@ public void testJson() {
         assertExpression("'{\"a\":1}' IS NOT JSON OBJECT").returns(false).check();
         assertExpression("'[1, 2]' IS NOT JSON ARRAY").returns(false).check();
         assertExpression("'1' IS NOT JSON SCALAR").returns(false).check();
+
+        // TODO https://issues.apache.org/jira/browse/IGNITE-20163
+        // assertExpression("'{\"a\":1}' FORMAT JSON").check();
+    }
+
+    @Test
+    @SuppressWarnings("ThrowableNotThrown")
```

Comment:
it looks like our gap in support of JSON. I suggest to create a ticket to support it, rewrite test  and mute it under the newly created ticket

Reply:
Fixed, thanks